cns-hook-util

```diff
@@ -6,6 +6,7 @@
 
 # Copyright 2021 Joyent, Inc.
 # Copyright 2023 MNX Cloud, Inc.
```

Comment:
MNX only needs one copyright line with the most recent year.

```
# Copyright 2021 Joyent, Inc.
# Copyright 2024 MNX Cloud, Inc.

```

Reply:
Done in 9813e31a2b1cff3cb07994bde4b5270ee6c8aaab.